dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -156,6 +156,10 @@ JSON_FIELDS_INVALID_CHARACHTER=TRAS3011W: An invalid character was provided in t
 JSON_FIELDS_INVALID_CHARACHTER.explanation= Only letters, numbers, and : are accepted in the JSON logging field name mapping property
 JSON_FIELDS_INVALID_CHARACHTER.useraction= Change your JSON logging field name mapping property value to remove invalid characters
 
+JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED=TRAS3012W: Header name was not specified for accessLogging logFormat field(s): {0}.
```

Comment:
Rather than "field(s)," use "fields" or add "one or more" to the sentence. Suggested rewrites:
> Header name was not specified for accessLogging logFormat fields: {0}.

> Header name is missing for one or more accessLogging logFormat fields: {0}.

Reply:
I'll address these comments in a follow-up PR so that someone else can deliver their code (which has merge conflicts with mine). Thanks for the review!